src/braket/aws/aws_quantum_task.py

```diff
@@ -522,7 +519,7 @@ def _create_internal(
     gate_definitions: Optional[dict[tuple[Gate, QubitSet], PulseSequence]],
     *args,
     **kwargs,
-) -> AwsQuantumTask:
+) -> AwsQuantumTask:  # ARG001
```

Comment:
Is there any way to specify this check (ignore dispatch methods) etc. Or would it be proper to narrow the signature of this method?

Reply:
None that I am aware of but I am researching more options.